varcode/vcf.py

```diff
@@ -363,9 +363,15 @@ def read_vcf_into_dataframe(path, include_info=False, chunk_size=None):
     else:
         raise NotImplementedError("Only local files are supported.")
 
+    compression = None
+    if path.endswith(".gz"):
+        compression = "gzip"
+    elif path.endswith(".bz2"):
+        compression = "bzip2"
```

Comment:
I think we want `bz2` here, 

From the Pandas doc: `compression : {‘gzip’, ‘bz2’, None}, default None`